bands/models.py

```diff
@@ -122,7 +122,7 @@ class BandTaxes(MagModel):
 
     @property
     def w9_url(self):
-        return '{}/bands/view_w9?id={}'.format(c.URL_BASE, self.band.id) if self.completed_w9 else ''
+        return '../bands/view_w9?id={}'.format(self.band.id) if self.w9_filename else ''
```

Comment:
also removed c.URL_BASE which we should avoid using unless it's 100% necessary (usually, just in email templates).  if we can get somewhere with a relative path, let's do that instead because otherwise we break accessing uber via proxies (like with Docker and more complex nging routing)